exporter/src/main/as/flump/export/AnimPreviewWindow.mxml

```diff
@@ -1,22 +1,18 @@
 <s:Window
     xmlns:fx="http://ns.adobe.com/mxml/2009"
-    xmlns:mx="library://ns.adobe.com/flex/mx"
     xmlns:s="library://ns.adobe.com/flex/spark"
     creationComplete="init();"
     alpha="0"
     renderMode="direct"
-    width="480" height="720">
+    width="480" height="720" showStatusBar="false">
```

Comment:
I complied it with Apache Flex 13 which added a bottom status bar to each window. This setting removes them